### ISSUE TITLE
Doesn't detect usage within an exec

### ISSUE THREAD

**Issue by tandrewnichols** [syngan/vim-vimlint#111]
E.g. I get "unused argument `a:def`" for this function:

```vim
function! s:Set(option, def)
  exec "let g:rebuff_" . a:option . " = get(g:, 'rebuff_" . a:option . "', a:def)"
endfunction
```
